MarsOnboardingTask/StepDefinitions/LanguagesStepDefinitions.cs

```diff
@@ -0,0 +1,112 @@
+using MarsOnboardingTask.Pages;
+using MarsOnboardingTask.Utils;
+using NUnit.Framework;
+
+
+namespace MarsOnboardingTask.StepDefinitions
+{
+    [Binding]
+    public class LanguagesStepDefinitions : Hooks
+    {
+        Login loginPageObj = new Login();
+        Languages languageObj = new Languages();
+
+        [Given(@"User logs into Mars application")]
+        public void GivenUserLogsIntoMarsApplication()
+        {
+            loginPageObj.LoginInToApplication(driver);
+        }
+
+        [When(@"User adds '([^']*)' language with '([^']*)' language level into the languages list")]
+        public void WhenUserAddsLanguageWithLanguageLevelIntoTheLanguagesList(string language, string languageLevel)
+        {
+            languageObj.AddLanguageWithlanguageLevel(driver, language, languageLevel);
+        }
+
+        [Then(@"The '([^']*)' language with '([^']*)' language level should be added to languages list")]
+        public void ThenTheLanguageWithLanguageLevelShouldBeAddedToLanguagesList(string language, string languageLevel)
+        {
+            string notification = languageObj.VerifytheLanguageAdded(driver, language, languageLevel);
+            String Message = language + " has been added to your languages";
+            String DuplicateMess = "Duplicated data";
+
+            if (notification == (language + " has been added to your languages"))
```

Comment:
you should not assert like this if not of the ifs execute then test will still pass! instead you should create multiple steps which should be clear what they assert! Please fix this everywhere

Reply:
updated as per the review comments